.github/workflows/test-command.yml

```diff
@@ -85,9 +85,6 @@ jobs:
         id: test
         env:
           ACTION_RUN_ID: ${{github.run_id}}
-          BUILD_STAT_WRITE_KEY: ${{ secrets.BUILD_STAT_WRITE_KEY }}
```

Comment:
these seemed to be unused

Reply:
forgot to remove them after switching to s3 for connector build history